### PR TITLE
Fixes bug with coordinator not filtering only electricity invoices

### PR DESCRIPTION
**Pull request by idoaflalo**
While the coordinator fetches the data from the API, it does not filter the documents related only to an electricity invoice, so for example, invoices related to service invoices are also present in the `billing_invoices.invoices` list. Thus, the plugin receives errors and displays incorrect information.

To solve this problem, I added a const named 'ELECTRICITY_INVOICE_ID', which is the electricity invoice ID (id=1), and added a filter after fetching the data. 

I have tested the code, and it works great 😊.

### CODE REVIEW

File: custom_components/iec/const.py [hunk 1]
```diff
@@ -33,3 +33,4 @@
 FUTURE_CONSUMPTIONS_DICT_NAME = "future_consumption"
 STATIC_KWH_TARIFF = "kwh_tariff"
 STATIC_BP_NUMBER = "bp_number"
+ELECTRICITY_INVOICE_ID = "1"
```
Comment (GuyKh) [inline custom_components/iec/const.py:36]: Please rename to `ELECTRIC_INVOICE_DOC_ID`
```suggestion
ELECTRIC_INVOICE_DOC_ID = "1"
```

**Comment by GuyKh** [comment]
LGTM 👍🏻 (other than some renaming)

Just out of curiosity, what other types of invoices (and their IDs) did you saw?

File: custom_components/iec/const.py (hunk 1 above)
Comment (idoaflalo) [reply custom_components/iec/const.py]: Done 👍🏻

**Comment by idoaflalo** [comment]
> LGTM 👍🏻 (other than some renaming)
> 
> Just out of curiosity, what other types of invoices (and their IDs) did you saw?

I called for a technician a month ago to fix a burned-out fuse in the street.
Following this, they added this invoice to my account as a document of the issue, LOL 🤷🏻‍♂️ .
image

**Comment by GuyKh** [comment]
Reverse engineering ids hahaha...

**Comment by GuyKh** [comment]
Thanks for the contribution!

**Pull request merged**